packages/commerce-sdk-react/src/api-client/pkce.js

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
we don't need any PKCE code as that's implemented in the commerce-sdk-isomorpic library. 

(In retail react app, we haven't refactor to use the SLAS helpers but we should do that in our hook library)